thumbfast.lua

```diff
@@ -272,7 +272,10 @@ end
 local function calc_dimensions()
     local width = mp.get_property_number("video-out-params/dw")
     local height = mp.get_property_number("video-out-params/dh")
-    if not width or not height then return end
+    if not width or not height then
+        effective_w, effective_h = 0, 0
```

Comment:
We can't do that, it snowballs into creating the following issues:
- vf_string adds `scale=w=0:h=0`
- the resulting thumbnail is at the original video size
- when entering check_new_thumb we now know the expected resize resolution (e.g. 200x150)
- but trying to run `real_res()` on 200x150 + an impossibly high filesize stat (image taken with previous "no scaling" vf) spits out a very wrong answer e.g. 13824x150

Reply:
If that wasn't clear, just remove the commit unless you happen to have a good reason to change the behavior here.